cmake/sdp.cmake

```diff
@@ -56,9 +60,9 @@ function(sdp_assembly_generate hrt_srcs)
     get_filename_component(src_filename ${hrt_src} NAME_WE)  # filename without extension
     add_custom_command(TARGET asm_gen
       PRE_BUILD
-      BYPRODUCTS ${src_filename}-temp.s
-      COMMAND ${CMAKE_C_COMPILER} ${compiler_options} ${hrt_opts} -S ${hrt_src} -o ${src_filename}-temp.s
-      COMMAND ${PYTHON_EXECUTABLE} ${ZEPHYR_NRF_MODULE_DIR}/scripts/sdp/remove_comments.py ${src_filename}-temp.s
+      BYPRODUCTS ${src_filename}-${target_soc}-temp.s
```

Comment:
`target_soc` is undefined if this function is called directly.

Reply:
Added as a parameter, is that okay or further validation is necessary?